src/main/java/redis/clients/jedis/Jedis.java

```diff
@@ -1673,6 +1673,21 @@ public Double zscore(final String key, final String member) {
     return BuilderFactory.DOUBLE.build(client.getOne());
   }
 
+  @Override
+  public Tuple zpopmax(final String key) {
+    checkIsInMultiOrPipeline();
+    client.zpopmax(key);
+    List<byte[]> l = client.getBinaryMultiBulkReply();
+    return l.isEmpty() ? null : new Tuple(l.get(0), BuilderFactory.DOUBLE.build(l.get(1)));
```

Comment:
You've created `BuilderFactory.TUPLE`. Can't we use it here to avoid code repetition?